py3status/modules/battery_level.py

```diff
@@ -95,6 +104,8 @@
 FORMAT = u"{icon}"
 FORMAT_NOTIFY_CHARGING = u"Charging ({percent}%)"
 FORMAT_NOTIFY_DISCHARGING = u"{time_remaining}"
+SYS_BATTERY_PATH = u"/sys/class/power_supply/"
```

Comment:
This doesn't need the `u` however it doesn't do any harm.

Also I'm not sure why these constants are defined rather than just explicitly defined in the module like most other do.

This comment can be ignored it is more observations.